test/library/standard/List/perf/listBenchmark1.chpl

```diff
@@ -0,0 +1,172 @@
+//
+// A performance test benchmarking common list operations.
+//
+
+private use HaltWrappers;
+private use List;
+private use Random;
+private use Time;
+
+type byte = int(8);
+type testList = list(byte, false);
+
+config param isPerformanceTest: bool = false;
```

Comment:
Nit, but I'd probably make this a config const, and move setting it into the perfexecopts (lets you get rid of a file, and doesn't require recompilation when running manually.) There should be no overhead for doing this check at execution time

Reply:
Thanks, that's a great point!